r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -724,3 +724,67 @@ test_that("errors in strptime", {
     'Time zone argument not supported by Arrow'
   )
 })
+
+test_that("str_like", {
+  
+  df <- tibble(x = c("Foo and bar", "baz and qux and quux"))
+  
+  # This will give an error until a new version of stringr with str_like has been released
+  expect_error(
+      expect_dplyr_equal(
+        input %>%
+          mutate(x = str_like(x, "%baz%")) %>%
+          collect(),
+        df,
+      )
+  )
+  
+  # After new version of stringr with str_like has been released, update all these
+  # tests to use expect_dplyr_equal
```

Comment:
Add `TODO: ` in the comment text per convention.
```suggestion
  # TODO: After new version of stringr with str_like has been released, update all
  # these tests to use expect_dplyr_equal
```
I'd say this is a good example of a place where just a TODO comment like this is sufficient and it doesn't justify creating a Jira for this.